Weather App/model/Constants.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
```

Comment:
It will be better to have constants for each view, not global

Reply:
👍